qiskit/pulse/builder.py

```diff
@@ -731,17 +737,6 @@ def append_instruction(self, instruction: instructions.Instruction):
         """
         self._context_stack[-1].append(instruction)
 
-    @_compile_lazy_circuit_before
-    def append_block(self, context_block: ScheduleBlock):
```

Comment:
Are there any backwards compatibility implications with this removal. The documentation referred to this method previously so was it user facing or is this all internal?

Reply:
This should be okey because builder itself is a protected class.
https://github.com/Qiskit/qiskit-terra/blob/8bfb058b9d94353936d7e2aea61015095487b873/qiskit/pulse/builder.py#L520-L521